tests/test_visitors/test_ast/test_builtins/test_strings/test_overused_string.py

```diff
@@ -37,10 +37,11 @@ def test_string_overuse(
     default_options,
 ):
     """Ensures that over-used strings raise violations."""
-    tree = parse_ast_tree(string_actions.format(string_value))
+    for string_val in string_values:
```

Comment:
We use `@pytest.mark.parameterize` instead.
See: https://github.com/wemake-services/wemake-python-styleguide/blob/master/tests/test_visitors/test_ast/test_classes/test_class_attributes/test_slots_syntax.py#L21-L24

Docs: https://docs.pytest.org/en/latest/parametrize.html